scripts/replace_parameter.sh

```diff
@@ -1,27 +1,59 @@
 #!/bin/bash
 
 usage() {
-	echo './replace_parameter.sh <REGISTER> <VALUE> <LINK>'
+
+	echo './replace_parameter.sh [-f] <REGISTER> <LINK> <VALUE/FILENAME>'
 	echo ''
 	echo '	REGISTER - register to be updated dropping the "CFG_" substring'
 	echo ''
-	echo '	VALUE - value in decimal to be assigned to REGISTER'
+	echo '	VALUE/FILENAME - value in decimal to be assigned to REGISTER or, if the -f option is used, filename containing a list of vfat number/ value pairs'
 	echo ''
-	echo '	Example:'
+	echo '	Examples:'
 	echo ''
 	echo '		./replace_parameter.sh PULSE_STRETCH 4 1'
-	echo ''
-
-	kill -INT $$; 
+	echo '		./replace_parameter.sh -f PULSE_STRETCH 4 /path/to/NominalDacValues.txt'
 }
 
-if [ -z ${3+x} ]
-then
-	usage
-fi
-
-REGISTER=$1
-VALUE=$2
-LINK=$3
+ISFILE=0;
+OPTIND=1
+while getopts "fwhd" opts
```

Comment:
It seems you have some undocumented/unused flags here (`wd`)

Reply:
fixed